components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -80,23 +80,33 @@ public void execute() {
 
         String projectName = this.projectName.replaceAll("[/\\\\]", "");
         File projectLocation = new File(GatewayCmdUtils.getProjectDirectoryPath(projectName));
+        try {
+            String projectCanonicalPath = projectLocation.getCanonicalPath();
+            String projectAbsolutePath = projectLocation.getAbsolutePath();
 
-        if (!projectLocation.exists()) {
-            throw new CLIRuntimeException("Project " + projectName + " does not exist.");
-        }
-        //extract the ballerina platform and runtime
-        ToolkitLibExtractionUtils.extractPlatformAndRuntime();
+            if (!projectLocation.exists()) {
```

Comment:
In windows powershell if you type **pet** and hit `tab` to autocomplete **petstore** directory, autocompleted directory name will endup as **.\petstore** in that case this will complain project doesn't exists.

Reply:
lets fix this later after a discussion